mlonmcu/session/postprocess/postprocesses.py

```diff
@@ -242,8 +243,10 @@ def __init__(self, features=None, config=None):
 
     @property
     def mapping(self):
-        # TODO: allow passing via cmdline
-        return self.config["mapping"]
+        value = self.config["mapping"]
+        if not isinstance(value, dict):
```

Comment:
Assert `isinstance(value, str)` here or inside `str2dict`. (For `str2bool` We should do this as well)